.github/workflows/ci.yml

```
@@ -0,0 +1,51 @@
+name: Run in-toto tests and linter
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      # Run regular TUF tests on each OS/Python combination, plus linters on
+      # Linux/Python3.x only.
+      matrix:
+        python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        toxenv: [py]
+        include:
+          # TODO: Change to 3.x once pylint fully supports Python 3.9
+          - python-version: 3.8
+            os: ubuntu-latest
+            toxenv: lint
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout in-toto
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Find pip cache dir
+        id: pip-cache
+        run: echo "::set-output name=dir::$(pip cache dir)"
+
+      - name: pip cache
+        uses: actions/cache@v2
+        with:
+          # Use the os dependent pip cache directory found above
+          path: ${{ steps.pip-cache.outputs.dir }}
+          # A match with 'key' counts as cache hit
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements*.txt') }}
+          # A match with 'restore-keys' is used as fallback
+          restore-keys: ${{ runner.os }}-pip-
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install tox
+      - name: Run tox
+        run: tox -e ${{ matrix.toxenv }}
```

Please add a trailing newline:
```suggestion
        run: tox -e ${{ matrix.toxenv }}

```